umlfri2/plugin/communication/executor.py

```diff
@@ -2,7 +2,11 @@
 import traceback
 import sys
 from base64 import b64encode
-from collections import Iterable
+
+try:
+    from collections.abc import Iterable
+except ImportError:
```

Comment:
Fallback for the old import is not needed as all supported versions of Python 3 contains abc module.